frontend/src/api/authRequest.tsx

```diff
@@ -1,19 +1,22 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
 import {IUserAuthApiResponse, IUserDetails} from '../actions/authActions'
 import { HttpClient} from './HttpClient'
 
 
 export interface IAuthApiClient {
-    register: (payload: IUserDetails) => Promise<IUserAuthApiResponse>
-    login: (payload: IUserDetails) => Promise<IUserAuthApiResponse>
+    register: (payload: IUserDetails, useCredentials: true) => Promise<IUserAuthApiResponse>
```

Comment:
You don't have useCredentials argument in actual register and login methods.
Also, you don't need it.

Reply:
Dont quite understand how its working, but I removed and everything still works.